Tests/AlicerceTests/Logging/ColoredLevelTests.swift

```diff
@@ -90,16 +91,19 @@ class ColoredLevelTests: XCTestCase {
 
         queue.dispatchQueue.sync {
             let expected = "\u{1B}[38;5;251mverbose message\n\u{1B}[38;5;35mdebug message\n\u{1B}[38;5;38minfo message\n\u{1B}[38;5;178mwarning message\n\u{1B}[38;5;197merror message"
-            let content = self.logfileContent(logfileURL: logfileURL)
+
+            let (content, numberOfLines) = logfileContent(logfileURL: logfileURL)
             XCTAssertEqual(content, expected)
-            XCTAssertEqual(destination.writtenItems, 5)
+            XCTAssertEqual(numberOfLines, 5)
         }
     }
 
     //MARK:- private methods
 
-    private func logfileContent(logfileURL: URL) -> String {
+    private func logfileContent(logfileURL: URL) -> (String, Int) {
 
-        return (try? String(contentsOf: logfileURL)) ?? ""
+        let content = (try? String(contentsOf: logfileURL)) ?? ""
+        let numberOfLines = content.split(separator: "\n").count
+        return (content, numberOfLines)
```

Comment:
since the expected content already contains the `\n`'s, it should be enough to test the output, right?